src/locale/dim.json

```diff
@@ -314,7 +314,8 @@
     "Shop": "Shop",
     "SupportDIM": "Backers",
     "UpgradeDIM": "Update DIM",
-    "WhatsNew": "What's New"
+    "WhatsNew": "What's New",
+    "Menu": "Menu"
```

Comment:
Minor, but localization keys go in alphabetical order, right?

Reply:
That happens automatically when the linter runs.